src/js/Pane/Window.js

```diff
@@ -160,4 +162,6 @@ Wui.Window.prototype = $.extend(true, {}, Wui.Pane.prototype,{
                 },
     height:     300,   
     width:      400
-});
\ No newline at end of file
+});
+
+})(jQuery,window[_wuiVar]);
```

Comment:
Just a parting comment to reinforce that these wrappers ought to be removed from all files in this review.